.github/workflows/starship-tests.yml

```diff
@@ -0,0 +1,93 @@
+name: Starship E2E Tests
+# E2E tests using Starship, run on a schedule and on every push to main
+on:
+  schedule:
+    - cron: "0 */2 * * *" # every 2 hours
```

Comment:
can we do every 6 hours, 2 hours may pollute the ci and block otherthings. We dont merge things fast enough for every 2 hours